src/pages/Privacy/PopUp.js

```diff
@@ -67,27 +67,28 @@ const useStyles = makeStyles((theme) => ({
 const PopUp = () => {
   const classes = useStyles();
   const date = new Date();
-  const setCookie = function (cookieEmail) {
-    let cookieValue, expDays;
-    date.setTime(date.getTime() + expDays);
-    const expires = 'expires=' + date.toUTCString();
-
-    document.cookie = `${cookieEmail}${cookieValue}';'${expires}';path=/'`;
 
-    return document.cookie;
+  const setCookie = function (cookieEmail, cookieValue, expDays) {
+    date.setTime(date.getTime() + (expDays * 24 * 60 * 60 * 1000));
+    const expires = 'expires=' + date.toUTCString();
+    const cookie = `${cookieEmail}=${cookieValue};${expires}`;
+    document.cookie = cookie;
+    return cookie;
   };
 
   const deleteCookie = function (cookieEmail) {
```

Comment:
Some of this code might have been copied over from the 311 data project. We should probably update it to es2015 arrow function format for code consistency

```
const deleteCookie = (cookieEmail) => {
   ...
};
```